### PR TITLE
Update Documentation to Reflect Latest Changes, Demo Instructions

### LINKED ISSUE #485: Create a Meaningful, E2E Demo Of BeaconChain+Validator
Hi all,

This is an issue tracking progress towards creating a meaningful demo of our current beacon chain + validator system. Ideally, a user should be able to spin up a beacon node as a validator, connect one or more validator clients to it via RPC, and see meaningful interactions occurring.

Some of the Desired Features Are:
- [x] Initial + Incoming Block Sync
- [x] Validator Shuffling, State Transitions, Validator Being Selected On A Slot/Shard
- [x] Validator P2P Interactions (can all be in the same network for now)
- [x] Beacon Node and Validator Use Internal Clock for Slot Intervals
- [x] Attester Responsibility Implemented
- [x] --dev mode to stub out web3 service and signature verification for local demo
- [x] Stream Validator Assignments via gRPC to Validator Clients
- [x] Give Each Validator a Unique Identifier (PubKey) So We Can Attach 2+ Validators to Beacon Node
- [x] Flag for Different Config Values for Demo (CycleLength, etc.)
- [x] Leverage Simulator for Demo
- [ ] Update Documentation to Clearly Reflect the Demo Functionality

Let's tag this issue in any open PR's or new PR's moving forward. Once we get to the point where we can demo the system locally, we can close this one :).

### PR DESCRIPTION
**Pull request by rauljordan**
This is the final part that resolves #485.

---

# Description

This PR includes updates to the documentation required to run our demo as well as updates to cli flags that make it easy for people to try out the system.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,48 +55,7 @@ bazel build //beacon-chain:beacon-chain
...
-```
-bazel run //beacon-chain --\
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-```
-
-### Running via Docker
+## Running via Docker
 
 To run the beacon node within a docker container, use the `//beacon-chain:image` target.
```
Comment (prestonvanloon) [inline README.md]: We actually provide two docker images for public consumption:

gcr.io/prysmaticlabs/beacon-chain:latest
gcr.io/prysmaticlabs/validator:latest

Should we update these instructions to run without bazel, using docker only?

Comment (prestonvanloon) [reply README.md]: Note: I haven't pushed these up yet, but I will do it when we tag the 0.0.0 release.

Comment (rauljordan) [reply README.md]: Sure thing

File: README.md [hunk 2]
```diff
@@ -46,130 +46,68 @@ Bazel manages all of the dependencies for you (including go and necessary compil
...
 
-```text
-bazel run //beacon-chain:image --\
-  --web3provider  ws://127.0.0.1:8546 \
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-  --simulator \
-  --verbosity debug
-```
+Once your beacon node is up, you'll need to attach a validator client as a separate process. This validator is in charge of running Casper+Sharding responsibilities (shards to be designed in phase 2). This validator will listen for incoming beacon blocks and shard assignments and determine when its time to perform attester/proposer responsibilities accordingly.
```
Comment (terenc3t) [inline README.md]: shards to be designed in phase 1 https://github.com/ethereum/wiki/wiki/Sharding-roadmap

the difference is phase 1 is shards data layer, phase 2 is shards execution layer

**Pull request merged**